### PR TITLE
content(cms): update Resource "how-to-publish-your-dictionary-data-with-publex/index"

### PR DESCRIPTION
**Pull request by KleeAn**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by ttasovac** [comment]
@stefanprobst when it rains, it pours 😄 here's another CMS-generated PR that gives me a render error on Vercel.

File: content/posts/how-to-publish-your-dictionary-data-with-publex/index.mdx [hunk 1]
```diff
@@ -362,12 +352,12 @@ There are a few basic things to keep in mind when defining the styling rules:
...
-  font](https://tcdh.uni-trier.de/en/projekt/komplett), as it contains more
-  characters compared to other fonts and, therefore, the probability is higher
-  that all characters can be displayed.
-</SideNote>
+
+>   It is recommended to use the [KompLett
+>   font](https://tcdh.uni-trier.de/en/projekt/komplett), as it contains more
+>   characters compared to other fonts and, therefore, the probability is higher
+>   that all characters can be displayed.
+> </SideNote>
```
Comment (stefanprobst) [inline content/posts/how-to-publish-your-dictionary-data-with-publex/index.mdx]:
```suggestion
>
  It is recommended to use the [KompLett
  font](https://tcdh.uni-trier.de/en/projekt/komplett), as it contains more
  characters compared to other fonts and, therefore, the probability is higher
  that all characters can be displayed.
</SideNote>
```

File: content/posts/how-to-publish-your-dictionary-data-with-publex/index.mdx [hunk 2]
```diff
@@ -481,9 +471,9 @@ After clicking on "OK", the rules are imported and displayed.
 <SideNote
   type="warning"
   title="Saving"
->
-  Make sure to click on "Save" after importing the rules.
-</SideNote>
+
+>   Make sure to click on "Save" after importing the rules.
+> </SideNote>
```
Comment (stefanprobst) [inline content/posts/how-to-publish-your-dictionary-data-with-publex/index.mdx]:
```suggestion
>
  Make sure to click on "Save" after importing the rules.
</SideNote>
```

**Comment by ttasovac** [comment]
Thanks a lot @stefanprobst

**Pull request merged**